app/javascript/components/enrollment/steps/Identification.js

```diff
@@ -23,6 +23,7 @@ class Identification extends React.Component {
       errors: {},
       modified: {},
       languageOptions: this.getLanguageOptions(),
+      age: '',
```

Comment:
This is patient-specific data so why would it need a separate state field? I would prefer keeping it how it was previously if possible.

Reply:
The problem was that react was complaining that this field went from an uncontrolled to a controlled input because `age` is not a field of patient.

The problem with the way it was before (`value={this.state.current.patient.age || ''}`) is that whenever the age was 0, it would show up as blank instead, so I tried something like (`value={this.state.current.patient.age == null ? '' : this.state.current.patient.age}`) instead, which resulted in the react warning/error mentioned above

Reply:
Because `age` is actually saved in the patient table as an integer, this will still need to be updated. I'm going to make a separate ticket for making sure age is always calculated and not in the database.